### ISSUE TITLE
Support for Poco x5 PRO

### ISSUE THREAD

**Issue by RIllEX0** [edk2-porting/edk2-msm#318]
Please add support for Poco X5 PRO 5G Redwood
8/256 (snapdragon 778G, 8 cores)
Please, i want to test windows 11 on fast and strong phone, but i cant because of no support/ no unofficial too
My discord (Type me in discord for any answers in this issue, please🤥): `nxori_`

**Comment by nedokaka** [comment]
@RIllEX0 No.

**Comment by RIllEX0** [comment]
why😟

**Comment by nedokaka** [comment]
@RIllEX0 Because i said no

**Comment by halal-beef** [comment]
@RIllEX0

**Comment by RIllEX0** [comment]
yes? @halal-beef

**Comment by nedokaka** [comment]
@halal-beef

**Comment by nedokaka** [comment]
@nedokaka Xd

**Comment by JSPiRiT1337** [comment]
soon, almost ready

**Comment by RIllEX0** [comment]
> soon, almost ready

Really?!? Thank you so much, if this real

**Comment by rxuglr** [comment]
This is fake

**Comment by RIllEX0** [comment]
@JSPiRiT1337 How much soon? like a week? thanks for answering

**Comment by halal-beef** [comment]
fake

**Comment by rxuglr** [comment]
real

**Comment by koke228** [comment]
halal beef nigger

**Comment by halal-beef** [comment]
@koke228 LMAO

**Comment by JSPiRiT1337** [comment]
@halal-beef lmao dude just wait a couple of days and you'll see

**Comment by nedokaka** [comment]
@koke228

**Comment by RIllEX0** [comment]
bruh, random russian Илья, stop being dumbass, i asked for SMART question  and you just spam in issue
go touch grass nig-

**Comment by rxuglr** [comment]
@RIllEX0 Poco x5 pro

**Comment by halal-beef** [comment]
@rxuglr one must imagine Sisyphus oiled up

**Comment by RIllEX0** [comment]
@rxuglr yes, poco x5 pro, i guess this is strong phone, and windows gonna be flying on this phone without lags
but, who knows🤓

**Comment by nedokaka** [comment]
@RIllEX0

**Comment by rxuglr** [comment]
@RIllEX0 It cursed phone. Pharaohs curse

**Comment by halal-beef** [comment]
@rxuglr Return the slab

**Comment by rxuglr** [comment]
@halal-beef Who are you

**Comment by halal-beef** [comment]
@rxuglr Good risings Pharaoh

**Comment by RIllEX0** [comment]
One more tip to contributors and others, poco x5 pro already got unofficial, but stable twrp and ofox recovery (+ lot of custom roms)
soooo, just saying🤥

**Comment by halal-beef** [comment]
but my TV has mainline Linux

**Comment by rxuglr** [comment]
@RIllEX0 I'm gonna take away unofficial twrp and ofox from Poco x5 pro

**Comment by RIllEX0** [comment]
wdym?

**Comment by nedokaka** [comment]
@RIllEX0 Poco x3 pro:
395886 billion custom roms
Ofox,twrp,pbrp,shrp,fprb
Droidian, manjaro arm, sfos, ubt and windows
And broken mainline😁

**Comment by halal-beef** [comment]
he's saying he going to make it official :trollface:

**Comment by rxuglr** [comment]
@RIllEX0 You know what I mean☺️

**Comment by RIllEX0** [comment]
@nedokaka i meant, poco x5 pro, is new phone, lot of people think about that theres small count of recoverys and roms, so just saying

**Comment by rxuglr** [comment]
@RIllEX0  poco x5 pro = outdated phone

**Comment by halal-beef** [comment]
@rxuglr Poco X5 pro = modified Poco X3 nfc

**Comment by rxuglr** [comment]
@halal-beef poco = poor
